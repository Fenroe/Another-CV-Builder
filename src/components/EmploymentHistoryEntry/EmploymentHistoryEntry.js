function EmploymentHistoryEntry() {
    return (
        <div>
            <fieldset>
                <legend>Employment History</legend>
                <button>Add Employer</button>
            </fieldset>
        </div>
    )
};

export default EmploymentHistoryEntry;
