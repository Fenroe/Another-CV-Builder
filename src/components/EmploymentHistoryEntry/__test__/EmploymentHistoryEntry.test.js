import { render, cleanup } from "@testing-library/react";
import React from "react";
import "@testing-library/jest-dom";
import EmploymentHistoryEntry from "../EmploymentHistoryEntry";

afterEach(cleanup);

it('renders', () => {
    const {asFragment} = render(<EmploymentHistoryEntry />);
    expect(asFragment()).toMatchSnapshot();
});
