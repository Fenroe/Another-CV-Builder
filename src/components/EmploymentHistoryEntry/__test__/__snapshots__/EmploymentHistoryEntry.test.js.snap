// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <fieldset>
      <legend>
        Employment History
      </legend>
      <button>
        Add Employer
      </button>
    </fieldset>
  </div>
</DocumentFragment>
`;
